remoterover.sh

```diff
@@ -280,13 +289,19 @@ if [[ "x${RUN_SERIAL}" == "x1" ]]; then
   ) || die "Installing Minicom failed"
 fi
 
-if [[ "x${RUN_RASPICONF}" == "x1" ]]; then
-  info "Running raspi-config commands"
+if [[ "x${RUN_STM32}" == "x1" ]]; then
+  info "Enabling STM32"
   ( set -x
-  for cmd in "${RASPICMDS[@]}"; do
```

Comment:
Same here - why do we delete that?

Reply:
RASPICONF was moved up because expanding filesystem needs to be done first after booting the system. Main branch didn't have this change so it made this weird code chimera. If you look at line 250 everything is in place